jerry-core/ecma/operations/ecma-regexp-object.c

```diff
@@ -573,7 +573,7 @@ re_match_regexp (re_matcher_ctx_t *re_ctx_p, /**< RegExp matcher context */
           {
             JERRY_ASSERT (ecma_is_value_boolean (match_value));
             /* restore saved */
-            memcpy (re_ctx_p->saved_p, saved_bck_p, size);
+            memcpy ((void *) (re_ctx_p->saved_p), saved_bck_p, size);
```

Comment:
I'm a bit worried that this patch would only hide things. `re_ctx_p` is a pointer to `struct re_matcher_ctx_t`, and `saved_p` is `const lit_utf8_byte_t **saved_p`, i.e., a (non-const) pointer to a (non-const) pointer to a `const lit_utf8_byte_t`. If anything, the first parameter should be cast to `(lit_utf8_byte_t **) re_ctx_p->saved_p` but actually I'm not even sure why MSVC warns about this: the pointers in the array should not be treated as constant (if I'm not mistaken).